server/base/src/main/java/org/apache/accumulo/server/ServerOpts.java

```diff
@@ -27,6 +28,17 @@ public class ServerOpts extends ConfigOpts {
   @Parameter(names = {"-a", "--address"}, description = "address to bind to")
   private String address = null;
 
+  @Parameter(required = false, names = {"-g", "--group"},
```

Comment:
Adding this option to server types where it does not do anything seems confusing to me.  Like I can start a manager with the option, but it will not have any effect.  Scoping it to only the tserver and scan server seems less confusing to me.

Reply:
My comment above regarding using our normal configuration property would make this a non-issue. Only server types where it makes sense would have that configuration property.